Vostok.Hosting/Components/SystemMetrics/SystemMetricsBuilder.cs

```diff
@@ -77,9 +82,11 @@ private void RegisterProcessMonitor(SystemMetricsSettings settings, BuildContext
                 };
 
                 var collector = new CurrentProcessMetricsCollector(collectorSettings);
-                
-                collector.ReportMetrics(metricContext, settings.ProcessMetricsReportingPeriod);
+                context.DisposableHostExtensions.Add(collector.ReportMetrics(metricContext, settings.ProcessMetricsReportingPeriod));
```

Comment:
These lines will result in "Dispose of `CurrentProcessMetricsCollector` component..." in logs.
I don't think that's a problem though...